msteams-platform/sbs-tabs-and-messaging-extensions-with-sso.yml

```diff
@@ -38,17 +38,20 @@ items:
      | &nbsp; | [Microsoft 365 developer account](/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant) | Access to Teams account with the appropriate permissions to install an app. |
      | &nbsp; | [.NET Core SDK](https://dotnet.microsoft.com/en-us/download) | Customized bindings for local debugging and Azure Functions app deployments. If you haven't installed the latest version, install the portable version. |
```

Comment:
We can add 2 button on this page:
![image](https://github.com/user-attachments/assets/b0e1722f-1bba-4fff-bfff-3f6baa54367d)
https://review.learn.microsoft.com/en-us/microsoftteams/platform/sbs-tabs-and-messaging-extensions-with-sso?branch=pr-en-us-12246&tabs=ngrok%2Cvs2019&tutorial-step=4